src/Service/PaymentMethodService.php

```diff
@@ -185,6 +193,12 @@ public function getMethodsForCheckout()
             }
         }
 
+        foreach ($methods as $method) {
+            if(isset($lang[$method['method_name']])) {
+                $method['method_name'] = $lang[$method['method_name']];
+            }
+        }
+
```

Comment:
I would recommend to createUtility class called  `PaymentMethod::getWithTranslatedNames()` or something